server/src/main/java/com/netflix/conductor/bootstrap/ModulesProvider.java

```diff
@@ -119,6 +121,13 @@ protected void configure() {
                     bind(ExternalPayloadStorage.class).to(S3PayloadStorage.class);
                 }
             });
+        } else if (externalPayloadStorageType == ExternalPayloadStorageType.AZURE_BLOB) {
```

Comment:
Please refactor to move the Azure implementation to a module which can then be injected when required.